server/rosetta/converter.go

```diff
@@ -242,30 +242,6 @@ func (c converter) Meta(msg sdk.Msg) (meta map[string]interface{}, err error) {
 // as metadata
 func (c converter) Ops(status string, msg sdk.Msg) ([]*rosettatypes.Operation, error) {
 	opName := proto.MessageName(msg)
```

Comment:
@fdymylja With https://github.com/cosmos/cosmos-sdk/pull/9139#issuecomment-827503574, we're using `sdk.MsgTypeURL` everywhere. However, I'm still not clear if in rosetta we should include the leading `/` or not.

I would argue to use `sdk.MsgTypeURL` here (and change the node data too), is that fine by you?

Reply:
Sounds good to me, I don't have a strong opinion.

The identifier of the message is used for operation.Type field, so I think it's fine if it is with a slash.